src/program/snabb_lwaftr/tests/data/binding.table

```diff
@@ -1,7 +1,13 @@
 {
- {'127:2:3:4:5:6:7:128', '178.79.150.233', 1, 100, '8:9:a:b:c:d:e:f'},
- {'127:11:12:13:14:15:16:128', '178.79.150.233', 101, 64000},
- {'127:22:33:44:55:66:77:128', '178.79.150.15', 5, 7000},
- {'127:24:35:46:57:68:79:128', '178.79.150.2', 7800, 7900, '1E:1:1:1:1:1:1:af'},
- {'127:14:25:36:47:58:69:128', '178.79.150.3', 4000, 5050, '1E:2:2:2:2:2:2:af'}
+ {'127:2:3:4:5:6:7:128', '178.79.150.233', {80, 0, 16}, '8:9:a:b:c:d:e:f'},
+ {'127:11:12:13:14:15:16:128', '178.79.150.233', {2300, 0, 16}},
```

Comment:
I think we don't need to force a=0, given that we already have to look up the.... the what?  You see, we need names :)  I'll call it the bitwidth then.  "PSID bitwidth" -- how many bits wide the PSID is.  Anyway given that we have to look it up in a table, we might as well look up the shift at the same time.

We could alternately assume that shift = 16-bitwidth.  Probably a better way to do things.